node/src/BaseClient.ts

```diff
@@ -3494,6 +3495,36 @@ export class BaseClient {
         return this.createWritePromise(createObjectRefcount(key));
     }
 
+    /**
+     * Copies the value stored at the `source` to the `destination` key if the `destination` key does not
+     * yet exist.
+     *
+     * See https://valkey.io/commands/copy/ for more details.
+     *
+     * since Valkey version 6.2.0.
+     *
+     * @remarks When in cluster mode, `source` and `destination` must map to the same hash slot.
+     * @param source - The key to the source value.
+     * @param destination - The key where the value should be copied to.
+     * @param replace - If the destination key should be removed before copying the value to it.
+     * @returns `true` if `source` was copied, `false` if the `source` was not copied.
+     *
+     * @example
+     * ```typescript
+     * const result = await client.copy("set1", "set2", true);
+     * console.log(result); // Output: true - "set1" was copied to "set2".
+     * ```
+     */
+    public async copy(
```

Comment:
I'm not sure if we want this in GlideClusterClient.ts or here. In Java we have it in GenericBaseCommands which aligns with what we have here, in Python we have it in cluster_commands.py which does not align with what we have here. Maybe we should check with the team?

Reply:
@Yury-Fridlyand  what's thought on this?

Reply:
Argree to split and move to 2 client classes